File: remind-me-query/src/main/java/com/dutra/remind/me/query/CassandraConfig.java
package com.dutra.remind.me.query;

import org.springframework.context.annotation.Configuration;
import org.springframework.data.cassandra.config.AbstractReactiveCassandraConfiguration;
import org.springframework.data.cassandra.config.SchemaAction;
import org.springframework.data.cassandra.repository.config.EnableReactiveCassandraRepositories;

@Configuration
@EnableReactiveCassandraRepositories
public class CassandraConfig extends AbstractReactiveCassandraConfiguration {

	@Override
	protected String getKeyspaceName() {
		return "remind_me";
	}

	@Override
	protected int getPort() {
		return 9042;
	}

	@Override
	protected String getContactPoints() {
		return "127.0.0.1";
	}

	@Override
	public SchemaAction getSchemaAction() {
		return SchemaAction.CREATE_IF_NOT_EXISTS;
	}

	@Override
	public String[] getEntityBasePackages() {
		return new String[]{"com.dutra.remind.me.query"};
	}
}

File: remind-me-query-kotlin/src/main/kotlin/com/dutra/remind/me/query/CassandraConfig.kt
package com.dutra.remind.me.query

import org.springframework.context.annotation.Configuration
import org.springframework.data.cassandra.config.AbstractReactiveCassandraConfiguration
import org.springframework.data.cassandra.config.SchemaAction
import org.springframework.data.cassandra.repository.config.EnableReactiveCassandraRepositories

@Configuration
@EnableReactiveCassandraRepositories
class CassandraConfig : AbstractReactiveCassandraConfiguration() {

    override fun getKeyspaceName(): String {
        return "remind_me"
    }

    override fun getPort(): Int {
        return 9042
    }

    override fun getContactPoints(): String {
        return "127.0.0.1"
    }

    override fun getSchemaAction(): SchemaAction {
        return SchemaAction.CREATE_IF_NOT_EXISTS
    }

    override fun getEntityBasePackages(): Array<String> {
        return arrayOf("com.dutra.remind.me.query")
    }
}
